### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by erikwanhainen** [emmyyin/yolov5#1]
- [x] Preprocess data
- [x] Set up Google cloud
- [x] Evaluate results on pre trained model with the original dataset
- [x] Train from scratch with custom data set but with the same architecture as the pre-trained model.
- [x] Change output layer to match custom dataset
- [x] Freeze every layer except output layer, train on custom dataset
     - Evaluate results
     - Switch to the original output layer and see if the model still can recognize the original dataset
- [x] Define backbone and head. Freeze backbone and train on custom dataset
     - Evaluate results
     - Switch to the original output layer and see if the model still can recognize the original dataset
- [ ] Add more layers, freeze the other layers and train on custom dataset
     - Evaluate results
     - Dropout? Freeze some layers randomly?

Data augmentation
- [x] Test YOLOv5 data augmentations method one at the time
- [ ] Run with all YOLOv5 types of data augmentation 
- [x] Combine transfer learning and data augmentation?
